src/components/panel/_panel.scss

```diff
@@ -3,13 +3,13 @@
 @include exports("panel") {
 
   .govuk-c-panel {
+    margin-bottom:  $govuk-spacing-scale-3;
```

Comment:
Very minor point (I'm surprised the linter didn't pick it up) but there's an extra space here.

Reply:
Done! Yes it's odd that the linter doesn't pick up on this type of spacing issues.